src/styles/LandingPage.module.css

```diff
@@ -0,0 +1,29 @@
+.LandingPage {
+  padding: 80px;
+  text-align: center;
+}
+.Landingh1 {
+  font-size: 70px;
+}
+.Landingh2 {
+  font-size: 40px;
+  margin-bottom: 60px;
+}
+.LandingLink {
+  font-size: 24px;
+  text-decoration: none;
+  color: var(--main-font-color);
+  background-color: var(--secondary-color);
+  font-family: var(--main-font);
+  border: 1px solid black;
+  border-color: var(--background-color);
```

Comment:
when you're using the combined `border` syntax, it inherently combines several different properties like `border-width` `border-style` and `border-color`.

So you can remove line `19` and replace the `black` on line `18` with this color variable